### DISCUSSION TITLE
resize images when upload to temp folder

### DISCUSSION THREAD

**Discussion by allaghi** [rahulhaque/laravel-filepond#52] [Q&A]
Hi everyone, 
Have a nice day,
I am wondering if is there any way to resize images that are uploaded into temp folder
Thanks in advance.

**Comment by stratboy** [comment]
As far as I know, you could use the filepond js library itself. Check the docs, there are official plugins for this task.

https://pqina.nl/filepond/docs/api/plugins/image-transform/

Also, Spatie Media Library can generate tranformed images. In this case it's not what you're asking for, but maybe it can help. Original images will be uploaded (but think that you can limit file size with filepond), and then x transformed copies will be generated by Spatie Media Library, in queue, based on your personal configurations.
